src/main/java/com/amazon/opendistroforelasticsearch/sql/executor/format/ElasticsearchErrorMessage.java

```diff
@@ -0,0 +1,68 @@
+/*
+ *   Copyright 2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ *   Licensed under the Apache License, Version 2.0 (the "License").
+ *   You may not use this file except in compliance with the License.
+ *   A copy of the License is located at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *   or in the "license" file accompanying this file. This file is distributed
+ *   on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ *   express or implied. See the License for the specific language governing
+ *   permissions and limitations under the License.
+ */
+
+package com.amazon.opendistroforelasticsearch.sql.executor.format;
+
+import com.amazon.opendistroforelasticsearch.sql.utils.StringUtils;
+import org.elasticsearch.ElasticsearchException;
+import org.elasticsearch.action.search.SearchPhaseExecutionException;
+import org.elasticsearch.action.search.ShardSearchFailure;
+
+public class ElasticsearchErrorMessage extends ErrorMessage<ElasticsearchException> {
+
+    ElasticsearchErrorMessage(ElasticsearchException exception, int status) {
+        super(exception, status);
+    }
+
+    @Override
+    protected String fetchReason() {
+        return "Error occurred in Elasticsearch engine: " + exception.getMessage();
+    }
+
+    /** Currently Sql-Jdbc plugin only supports string type as reason and details in the error messages */
+    @Override
+    protected String fetchDetails() {
+        StringBuilder details = new StringBuilder();
+        if (exception instanceof SearchPhaseExecutionException) {
+            String detail = fetchSearchPhaseExecutionExceptionDetails((SearchPhaseExecutionException) exception);
+            details.append(detail);
```

Comment:
np: can be reduced to one line? `details.append(fetchSearchPhaseExecutionExceptionDetails((SearchPhaseExecutionException) exception);`

Reply:
yes, done now. thanks!